.github/actions/back/deploy/action.yml

```diff
@@ -42,14 +42,21 @@ runs:
         docker push "${{ steps.variables.outputs.image }}" -a
 
     - name: 🚀 Deploy to Cloud Run
-      uses: google-github-actions/deploy-cloudrun@v0.7.0
+      uses: google-github-actions/deploy-cloudrun@v0
       with:
         service: ${{ env.SERVICE_NAME }}
         image: ${{ steps.variables.outputs.image }}:latest
-        region: "us-central1"
+        region: us-central1
+        flags: |
+          --service-account=cloud-run@${{ env.PROJECT_ID }}.iam.gserviceaccount.com
+          --max-instances=10
+          --min-instances=${{ env.MIN_INSTANCES }}
+          --set-cloudsql-instances=app
+          --allow-unauthenticated
         env_vars: |
           ADMIN_EMAIL=${{ env.ADMIN_EMAIL }}
           API=${{ env.API }}
+          CLIENT_ID=${{ env.CLIENT_ID }}
           DATABASE_HOST=${{ env.DATABASE_HOST }}
           DATABASE_PASSWORD=${{ env.DATABASE_PASSWORD }}
           ENVIRONMENT=${{ env.ENVIRONMENT }}
```

Comment:
**issue**: most of these will need to be stored as secrets in the secret manager

Reply:
Is this required? I really don't want to tear apart my environment/secrets system at the moment. It took me a long time to get it organized and working. Is the issue that I shouldn't be storing secrets as environment variables? Or something else?

Reply:
Yes security called me out on it for uic and that is why I just went through the same process.

Reply:
This is ready for another look. I've also added the secrets to: https://github.com/agrc/gcp-terraform/pull/51